server/src/main.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+# Requires:  CherryPy 18.0.1  (www.cherrypy.org)
+#            Python  (We use 3.5.x +)
+
+import os
+
+import cherrypy
+
+import server
+
+import socket
+
+# The address we listen for connections on
+LISTEN_IP = "0.0.0.0"
+LISTEN_PORT = 1234
+
+def run_main_app():
+    #set up the config
+    #global LISTEN_IP
```

Comment:
these commented out code lines can be removed (and you have set them above anyway)